src/frontend/src/lib/components/tokens/TokensSignedIn.svelte

```diff
@@ -21,10 +22,38 @@
 		({ id: tokenId }) =>
 			($balancesStore?.[tokenId]?.data ?? BigNumber.from(0n)).gt(0n) || displayZeroBalance
 	);
+
+	let sortingEnabled = true;
```

Comment:
I did not understand this at first. Can we add a comment or maybe rename `pointerEventsStore`? The latter seems like a technical name from a store but without purpose WDYT?

Reply:
refactored a bit, this variable was deprecated